data/core/units/monsters/Jinn.cfg

```diff
@@ -101,7 +108,7 @@ units/monsters/jinn#enddef
         icon=attacks/lightning.png
         type=fire
         range=ranged
-        damage=18
+        damage=20
```

Comment:
20 seems really high… it _is_ a level 2 unit but I wonder if it might be better to split this damage across two strikes?

Reply:
I was thinking about it but after reading comment about animation I decided to leave it as one.